src/languages/es.ts

```diff
@@ -1343,7 +1347,7 @@ export default {
     },
     newChatPage: {
         createChat: 'Crear chat',
-        createGroup: 'Crear grupo',
+        startGroup: 'Grupo de inicio',
```

Comment:
This translation is wrong. I search in slack and I could not find any mention of this particular string, so I think it was not reviewed by anyone and if that's correct it means that @waterim you checked this checklist item without actually doing it:
>  If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:

Reply:
BTW changed this here https://github.com/Expensify/App/pull/45342